setup.cfg

```diff
@@ -0,0 +1,2 @@
+[metadata]
+description-file = README.md
```

Comment:
If we decide to use a `setup.cfg`, there are many things from the `setup.py` that should be move over here.